hail/python/hail/experimental/vcf_combiner/__main__.py

```diff
@@ -20,9 +20,19 @@ def main():
                              'WARNING: if this option is used, the sample names in the '
                              'GVCFs will be overridden by the names in sample map.',
                         required=False)
+    parser.add_argument('--genomes', dest='is_genomes', action='store_true',
```

Comment:
should we require the user to add this flag? That seems like a good idea to avoid people unintentionally using exome intervals for genome data.

Reply:
The question to me is what we would like the defaults to be. I thought about having genomes be the default, but decided against it while making this change. I have no particular reason or justification for doing so.

The reason I added both `--genomes` and `--exomes` was to better allow users to indicate their intentions in scripts.

Thoughts on the default?

Reply:
sorry, wasn't clear -- I'm proposing that we don't actually have a default, but instead require that one of the flags is passed explicitly by the user.

Reply:
For reference, here's what changed between the force pushes:
```diff
diff --git a/hail/python/hail/experimental/vcf_combiner/__main__.py b/hail/python/hail/experimental/vcf_combiner/__main__.py
index 9a5847372..91092add2 100644
--- a/hail/python/hail/experimental/vcf_combiner/__main__.py
+++ b/hail/python/hail/experimental/vcf_combiner/__main__.py
@@ -14,24 +14,25 @@ def main():
     parser.add_argument('out_file', help='Path to final combiner output.')
     parser.add_argument('tmp_path', help='Path to folder for intermediate output '
                                          '(should be an object store path, if running on the cloud).')
-    group = parser.add_mutually_exclusive_group(required=True)
-    group.add_argument('--genomes', '-G', dest='is_genomes', action='store_true',
-                       help='Indicates that the combiner is operating on genomes. '
-                            'Affects how the genome is partitioned on input.')
-    group.add_argument('--exomes', '-E', dest='is_genomes', action='store_false',
-                       help='Indicates that the combiner is operating on exomes. '
-                            'Affects how the genome is partitioned on input.')
-    group.add_argument('--import-interval-size', type=int,
-                       help='Interval size for partitioning the reference genome for GVCF import.')
     parser.add_argument('--log', help='Hail log path.')
     parser.add_argument('--header',
                         help='External header, must be readable by all executors. '
                              'WARNING: if this option is used, the sample names in the '
                              'GVCFs will be overridden by the names in sample map.',
                         required=False)
+    parser.add_argument('--genomes', dest='is_genomes', action='store_true',
+                        help='Indicates that the combiner is operating on genomes. '
+                             'Affects how the genome is partitioned on input. '
+                             'Ignored if --import-interval-size is used')
+    parser.add_argument('--exomes', dest='is_genomes', action='store_false',
+                        help='Indicates that the combiner is operating on exomes '
+                             '(This is the default). '
+                             'Affects how the genome is partitioned on input. '
+                             'Ignored if --import-interval-size is used')
     parser.add_argument('--branch-factor', type=int, default=CombinerConfig.default_branch_factor, help='Branch factor.')
     parser.add_argument('--batch-size', type=int, default=CombinerConfig.default_batch_size, help='Batch size.')
     parser.add_argument('--target-records', type=int, default=CombinerConfig.default_target_records, help='Target records per partition.')
+    parser.add_argument('--import-interval-size', type=int, help='Interval size for partitioning the reference genome for GVCF import.')
     parser.add_argument('--overwrite', help='overwrite the output path', action='store_true')
     parser.add_argument('--key-by-locus-and-alleles', help='Key by both locus and alleles in the final output.', action='store_true')
     parser.add_argument('--reference-genome', default='GRCh38', help='Reference genome.')
```